docker/scripts/start-light-core.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+echo "Install dependencies"
+# If no dependencies have been update, you can comment this line
+# to use up to 30sec in load time
+yarn install
+
+# Migrate database if needed (usually required if you run it for the first time)
+# sleep 30 # required to wait for the database to start
+# yarn run db:migrate
```

Comment:
Désolé de faire encore un retour tardif
Il faut jouer le `db:migrate` à chaque lancement. Il va vérifier si des changements en base sont nécessaires suite aux évolutions du code. 
Et le `sleep 30` n'est pas nécessaire si on utilise `wait-for-it` dans le fichier docker-compose

Reply:
Ok je viens de mettre à jour le script @TdyP